build.gradle

```diff
@@ -125,67 +132,66 @@ minecraft {
 	}
 }
 
-// prevent java 8's strict doclint for javadocs from failing builds
-if (JavaVersion.current().isJava8Compatible()) {
-	tasks.withType(Javadoc) {
-		options.addStringOption('Xdoclint:none', '-quiet')
-	}
+javadoc {
+	source = [sourceSets.main.allJava, sourceSets.api.allJava]
+	// prevent java 8's strict doclint for javadocs from failing builds
+	options.addStringOption('Xdoclint:none', '-quiet')
 }
 
 task makeChangelog(type: GitChangelogTask) {
-	file = new File("changelog.html")
+	file = file('changelog.html')
 	untaggedName = "Current release ${project.version}"
-	fromCommit = "2fe051cf727adce1be210a46f778aa8fe031331e"
-	toRef = "HEAD"
-	templateContent = file("changelog.mustache").getText("UTF-8")
+	fromCommit = '2fe051cf727adce1be210a46f778aa8fe031331e'
+	toRef = 'HEAD'
+	templateContent = file('changelog.mustache').getText('UTF-8')
 }
 
 curseforge {
-	apiKey = project.hasProperty("curseforge_apikey") ? project.curseforge_apikey : "0"
+	apiKey = project.findProperty('curseforge_apikey') ?: '0'
 	project {
 		id = curse_project_id
-		changelog = file("changelog.html")
-		changelogType = "html"
-		releaseType = "beta"
+		changelog = file('changelog.html')
+		changelogType = 'html'
+		releaseType = 'beta'
 	}
 }
 
 afterEvaluate {
-	tasks.curseforge238222.dependsOn.add(makeChangelog)
+	tasks.curseforge238222.dependsOn makeChangelog
 }
 
 jar {
-	from sourceSets.main.output.classesDirs
-	from sourceSets.api.output.classesDirs
-	from sourceSets.main.output.resourcesDir
-	from sourceSets.api.output.resourcesDir
-	
+	from sourceSets.main.output
+	from sourceSets.api.output
+
 	manifest {
 		attributes([
-				"Specification-Title": "Just Enough Items",
-				"Specification-Vendor": "mezz",
-				"Specification-Version": "${semver}",
-				"Implementation-Title": project.name,
-				"Implementation-Version": "${project.version}",
-				"Implementation-Vendor" :"mezz",
-				"Implementation-Timestamp": new Date().format("yyyy-MM-dd'T'HH:mm:ssZ"),
+				'Specification-Title'     : 'Just Enough Items',
+				'Specification-Vendor'    : 'mezz',
+				'Specification-Version'   : "${semver}",
+				'Implementation-Title'    : project.name,
+				'Implementation-Version'  : "${project.version}",
+				'Implementation-Vendor'   : 'mezz',
+				'Implementation-Timestamp': new Date().format('yyyy-MM-dd\'T\'HH:mm:ssZ'),
```

Comment:
this one actually seems like an appropriate use for `"` unless they've been completely disallowed